src/app/(dashboard)/teams/[teamId]/settings/page.tsx

```diff
@@ -0,0 +1,162 @@
+"use client";
+
+import { RegisteredGitRepo } from "@/app/api/github/repo/registered/fetchRegisteredRepos";
+import { GithubRepoDto } from "@/app/api/github/repo/types/githubRepo.dto";
+import BreadCrumb from "@/components/breadcrumbs";
+import { Heading } from "@/components/ui/heading";
+import { Separator } from "@/components/ui/separator";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { GitRepoView, columns } from "../../new/columns";
+import { DataTable } from "../../new/data-table";
+import { Button } from "@/components/ui/button";
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "@/components/ui/alert-dialog";
+import { useRouter } from "next/navigation";
+
+interface PageProps {
+  params: { teamId: string };
+}
+
+export default function TeamSettingsPage({ params }: PageProps) {
+  const teamId = params.teamId;
+  const breadcrumbItems = [
+    { title: "Teams", link: "/teams" },
+    { title: "Team details", link: `/teams/${teamId}` },
+    { title: "Team settings", link: `/teams/${teamId}/settings` },
+  ];
+
+  const router = useRouter();
+
+  const [registeredGitRepos, setRegisteredGitRepos] = useState<
+    RegisteredGitRepo[]
+  >([]);
+  const [page, setPage] = useState<number>(1);
+  const [canNext, setCanNext] = useState<boolean>(false);
+  const [canPrevious, setCanPrevious] = useState<boolean>(false);
+  const [githubRepos, setGithubRepos] = useState<GithubRepoDto[]>([]);
+  const [githubRepoColumnData, setGithubRepoColumnData] = useState<
+    GitRepoView[]
+  >([]);
+
+  const handleQueryGithubRepos = async (page: number = 1) => {
+    const { data } = await axios.get(`/api/github/repo?page=${page}`);
+    if (data.success && data.gitRepos.length > 0) {
+      setGithubRepos(data.gitRepos);
+    }
+  };
+
+  const handleFetchRegisteredRepos = async () => {
+    const { data } = await axios.get(
+      `/api/github/repo/registered?team_id=${teamId}`,
+    );
+    if (data.success) {
+      setRegisteredGitRepos(data.registeredRepos);
+    }
+  };
+
+  const handleSave = async () => {
```

Comment:
Why are we not redirecting back to the team details page ? Make more sense no?

Reply:
The team is going to be calculating after saving, so we would only be showing the "calculating" component anyway, I thought it would be better to redirect the user to somewhere with more data. 
Happy to be challenged on this ✅